src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -7761,8 +7763,14 @@ namespace InternalHeatGains {
                     WAirIn = state.dataHeatBalFanSys->ZoneAirHumRat(NZ);
                 } else {
                     // TAirIn = TRoomAirNodeIn, according to EngineeringRef 17.1.4
-                    int ZoneAirInletNode = state.dataZoneEquip->ZoneEquipConfig(NZ).InletNode(1);
-                    TSupply = state.dataLoopNodes->Node(ZoneAirInletNode).Temp;
+                    int zoneEqIndex =
+                        DataZoneEquipment::GetControlledZoneIndex(state, state.dataHeatBal->Zone(state.dataHeatBal->ZoneITEq(Loop).ZonePtr).Name);
```

Comment:
Would be faster to save the zoneEqIndex in the ZoneITEq struct than to look it up each iteration for each IT equipment.

Reply:
It's definitely not a one-time thing.  I'll fix that (hopefully today) and make another commit (that will include getting rid of the =0 thing in the unit test as well).

Reply:
This now looks up and saves the value, avoiding doing this more than once per ZoneITEq.  Change has been committed, branch updated to develop, and all of that pushed so that CI will run.  Hopefully this resolves this.  Thanks!